include/elliptics/packet.h

```diff
@@ -747,7 +747,9 @@ struct dnet_io_attr
 	 */
 	uint64_t		total_size;
 
-	uint64_t		reserved1;
+	/* Combination of DNET_RECORD_FLAGS_* */
+	uint64_t		record_flags;
```

Comment:
It is completely different from `io->flags` because `io->flags` is about how concrete read/write/remove command should be done, but `record_flags` is about how the entry is stored in the blob.